package.json

```
@@ -31,6 +43,7 @@
         "@types/node": "^18.11.9",
         "@typescript-eslint/eslint-plugin": "^5.21.0",
         "@typescript-eslint/parser": "^5.21.0",
+        "esbuild": "0.16.10",
```

What about using the `0.17.x` ? It seems you are also missing the caret `^` in front of the version number.
(I am not sure about the implications)

```suggestion
        "esbuild": "^0.17.18",
```

---

Caret sign must be added, it is safe.

---

I haven't had time to review the breaking changes in 0.17. Due to that, and because build tools are a pain anyway, I'd rather leave the hardcoded version there for now.

We can look at bumping the version after it has been merged and released.